docs/docker.md

```diff
@@ -0,0 +1,17 @@
+# Dockerfiles
+
+Our test runners are deployed as docker images.
+
+- Your dockerfile should live at the root directory of your repository and it should be called `Dockerfile`
+- Your dockerfile should be the minimal needed for your test runners and create the minimal image needed for running the tests.
+  It should use alpine linux if possible.
+- The working directory should be `/opt/test-runner`
+- There should be a `/opt/test-runner/bin/run.sh` script that can be called with 3 parameters: the `exercise slug`, the path to the `solution folder`, and the path to the `output folder`.
+  For more information see [The Interface](./interface.md).
+- All changes to dockerfiles require a PR review from the @exercism/ops team.
+
+More background information and optional hints:

```

Comment:
Could you maybe also add an entry that containers won't have access to the network when running? And that only the input and output directories and `/tmp` directories can be written to?

This can be tested by passing the `--network none` and `--read-only` flags to `docker run`.